modules/nw-sriov-network-operator.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-network-operator_{context}"]
+
+= Configuring an SR-IOV Network Operator for virtual functions
+The Single Root I/O Virtualization (SR-IOV) network operator is used to allocate and configure Virtual Functions (VFs) from SR-IOV-supporting Physical Function NICs on the nodes.
+
+For more information on deploying the operator, see _Installing the SR-IOV Network Operator_.
+For more information on configuring an SR-IOV network device, see _Configuring an SR-IOV network device_.
+.Procedure
+
+There are some differences between running Data Plane Development Kit (DPDK) workloads on Intel VFs or Mellanox VFs. This section provides object configuration examples for both VF types.
```

Comment:
Should "Intel VFs or Mellanox VFs" be "Intel VFs and Mellanox VFs". My apologies if I overlooked this in my first review.

Reply:
Done.